views/assets/stylesheets/components/header.css

```diff
@@ -105,6 +101,33 @@ a.qor-slideout__fullscreen > i.tyk-icon+.tyk-icon {
 	}
 }
 
+.profile-details-container__list {
+  color: var(--default-text-color);
+  border-radius: var(--default-border-radius);
+  background: var(--white);
+  position: absolute;
+  top: 45px;
+  right: 45px;
+  z-index: 10;
+  visibility: hidden;
+}
+
+.profile-details-container__list > li {
+  padding-block: 10px;
+  padding-left: 20px;
+  padding-right: 50px;
```

Comment:
I don't think it is a good idea to mix the use of logical padding and physical padding. Unless this is intentional I would change this to use either only logical padding css properties, or only physical padding ones.